server/schedule/operator/operator.go

```diff
@@ -886,41 +881,41 @@ func interleaveStepGroups(a, b [][]OpStep, sizeHint int) []OpStep {
 }
 
 // CreateMovePeerOperator creates an operator that replaces an old peer with a new peer.
-func CreateMovePeerOperator(desc string, cluster Cluster, region *core.RegionInfo, kind OpKind, oldStore, newStore uint64, peerID uint64) (*Operator, error) {
-	removeKind, steps, err := removePeerSteps(cluster, region, oldStore, append(getRegionFollowerIDs(region), newStore))
+func CreateMovePeerOperator(desc string, cluster Cluster, region *core.RegionInfo, kind OpKind, oldStore uint64, peer *metapb.Peer) (*Operator, error) {
+	removeKind, steps, err := removePeerSteps(cluster, region, oldStore, append(getRegionFollowerIDs(region), peer.StoreId))
 	if err != nil {
 		return nil, err
 	}
-	st := CreateAddPeerSteps(newStore, peerID)
+	st := CreateAddPeerSteps(peer)
 	steps = append(st, steps...)
-	brief := fmt.Sprintf("mv peer: store %v to %v", oldStore, newStore)
+	brief := fmt.Sprintf("mv peer: store %v to %v", oldStore, peer.StoreId)
 	return NewOperator(desc, brief, region.GetID(), region.GetRegionEpoch(), removeKind|kind|OpRegion, steps...), nil
 }
 
 // CreateOfflinePeerOperator creates an operator that replaces an old peer with a new peer when offline a store.
-func CreateOfflinePeerOperator(desc string, cluster Cluster, region *core.RegionInfo, kind OpKind, oldStore, newStore uint64, peerID uint64) (*Operator, error) {
-	k, steps, err := transferLeaderStep(cluster, region, oldStore, append(getRegionFollowerIDs(region)))
+func CreateOfflinePeerOperator(desc string, cluster Cluster, region *core.RegionInfo, kind OpKind, oldStore uint64, peer *metapb.Peer) (*Operator, error) {
+	k, steps, err := transferLeaderStep(cluster, region, oldStore, getRegionFollowerIDs(region))
 	if err != nil {
 		return nil, err
 	}
 	kind |= k
-	st := CreateAddPeerSteps(newStore, peerID)
+	st := CreateAddPeerSteps(peer)
 	steps = append(steps, st...)
 	steps = append(steps, RemovePeer{FromStore: oldStore})
-	brief := fmt.Sprintf("mv peer: store %v to %v", oldStore, newStore)
+	brief := fmt.Sprintf("mv peer: store %v to %v", oldStore, peer.StoreId)
```

Comment:
why modify `brief`?

Reply:
It does not modify `brief` but only move the storeID to peer.